server/mediaserver_test.go

```diff
@@ -42,6 +43,29 @@ func setupServer() *LivepeerServer {
 	return S
 }
 
+func startAuthTestServer() {
+	mux := http.NewServeMux()
+	srv := &http.Server{
+		Addr:    "127.0.0.1:8939",
+		Handler: mux,
+	}
+
+	//Set the broadcast config for creating onchain jobs.
+	mux.HandleFunc("/auth", func(w http.ResponseWriter, r *http.Request) {
+		w.Write(nil)
+	})
+	mux.HandleFunc("/auth/emptymanifest", func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte("{\"manifestID\":\"\"}"))
+	})
+	mux.HandleFunc("/auth/xy", func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte("{\"manifestID\":\"xy\"}"))
+	})
+	mux.HandleFunc("/auth/invalidjson", func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte("{manifestID\":}"))
+	})
+	go srv.ListenAndServe()
```

Comment:
Spinning up a live HTTP server for testing can be unpredictable / problematic so it's best to avoid it if possible. It's not always possible, but here I think we can avoid it here we follow the format that's used in https://github.com/livepeer/go-livepeer/blob/90c10193860999497cef5f1031fec4890bb9f19e/server/handlers_test.go

Reply:
done